wordpress-mobile/WordPress-Android
# Update Gutenberg so show content structure information.

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/506

GB-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2380

~This PR just updates the GB reference to have the new bridge method that sends the content metrics data.~

Shows the content structure information of the post:

<img src="https://user-images.githubusercontent.com/651601/84710184-a1396780-af5b-11ea-842e-3f749d6b0ceb.png" width=300>

To test:
 - Start the app
 - Create a post
  - Write some content
 - Tap on the top right ... button
 - Check if the content structure option is available
 - Tap on the option
 - Check if a dialog presents the information.

PR submission checklist:
